### PR TITLE
Fix common, foreman tests

### PR DESCRIPTION
**Pull request by arkid15r**
## Purpose/Implementation Notes

Migrate common/migrations image to `dr_base`.
Move foreman tests to tests directory.
Add separate settings file (tests/settings.py).
Refactor import according to the new structure.

## Methods

NA

## Types of changes

What types of changes does your code introduce?
- Bugfix (non-breaking change which fixes an issue)

## Functional tests

NA

## Checklist
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Screenshots

NA

### CODE REVIEW

File: foreman/tests/foreman/management/commands/test_update_experiment_metadata.py [hunk 1]
```diff
@@ -82,53 +83,49 @@ def test_sra_experiment_missing_alternate_accession(self):
...
+        experiment.title = SurveyTestCase.BAD_TITLE
         experiment.save()
 
         # 2. Setup is done, actually run the command.
         command = Command()
         command.handle()
 
-        # Test that the title was fixed
+        # Test that the title was fixed.
+        experiment = Experiment.objects.get_or_create(accession_code=experiment.accession_code)[0]
```
Comment (davidsmejia) [inline foreman/tests/foreman/management/commands/test_update_experiment_metadata.py:123]:
```suggestion
        experiment, _ = Experiment.objects.get_or_create(accession_code=experiment.accession_code)
```

**Review by davidsmejia — CHANGES_REQUESTED** [review]
LGTM just some minor suggestions for improvement

File: foreman/data_refinery_foreman/foreman/management/commands/update_experiment_metadata.py [hunk 2]
```diff
@@ -70,20 +71,18 @@ def handle(self, *args, **options):
                         GeoSurveyor._apply_metadata_to_experiment(experiment, gse)
 
                     elif experiment.source_database == "ARRAY_EXPRESS":
-                        request_url = EXPERIMENTS_URL + experiment.accession_code
```
[the remark this answers is not in the document]
Comment (arkid15r) [reply foreman/data_refinery_foreman/foreman/management/commands/update_experiment_metadata.py:73]: I'm going to clean that up after the ArrayExpress migration is done.

File: foreman/data_refinery_foreman/surveyor/array_express.py [hunk 3]
```diff
@@ -40,222 +41,236 @@ class ArrayExpressSurveyor(ExternalSourceSurveyor):
...
+                elif subsection["type"] == "Assays and Data":
+                    array_designs = []
+                    for assays_data_subsection in subsection["subsections"]:
+                        if assays_data_subsection["type"] == "Array Designs":
+                            for link in assays_data_subsection["links"]:
+                                for link_entry in link:
+                                    array_designs.append(link_entry["url"])
+                    parsed_data["array_designs"] = array_designs
+
+        return parsed_data
```
[the remark this answers is not in the document]
Comment (arkid15r) [reply foreman/data_refinery_foreman/surveyor/array_express.py:86]: Done.

File: foreman/tests/foreman/management/commands/test_update_experiment_metadata.py (hunk 1 above)
Comment (arkid15r) [reply foreman/tests/foreman/management/commands/test_update_experiment_metadata.py:123]: Done

**Review by davidsmejia — APPROVED** [review]
LGTM!

**Pull request merged**